### PR TITLE
Specify log retention policy

### PR DESCRIPTION
**Pull request by OisinKyne**
Not sure but think this will change the default to reflect historical logs through restarts, need to confirm better

### CODE REVIEW

**Review by xenowits — APPROVED** [review]
Looks good!

File: docker-compose.yml [hunk 1]
```diff
@@ -2,6 +2,13 @@ version: "3.8"
 
 # Override any defaults specified by `${FOO:-bar}` in `.env` with `FOO=qux`.
 
+x-logging: &logging
+  logging:
+    driver: json-file
```
Comment (corverroos) [inline docker-compose.yml:7]: not sure about wrapping everything in another json layer is helpful, I would just use a `local` driver.

Comment (corverroos) [reply docker-compose.yml:7]: seems like `local` is the new preferred driver in any case, json-file is a legacy backwards compatibility thing. https://docs.docker.com/config/containers/logging/configure/

**Comment by OisinKyne** [comment]
Closing as `local` is now the better way to do things.

**Pull request closed without merging**